arch/arm/src/sama5/sam_tc.c

```diff
@@ -1451,7 +1452,8 @@ int sam_tc_divisor(uint32_t frequency, uint32_t *div, uint32_t *tcclks)
 
   if (tcclks)
     {
-      tmrinfo("return tcclks=%08lx\n", (unsigned long)TC_CMR_TCCLKS(ndx));
+      tmrinfo("return tcclks=%08" PRIx32 "\n",
```

Comment:
This place does not need PRIx32 macro since argument is casted to unsigned long

Reply:
seems very inconsistent to me to sometimes recast and sometimes use a macro?

Reply:
All is consistent. `uint32_t`/`int32_t` should be printed with `PRId/u/x32` macro while `long`/`unsigned long` should use `ld/u/x` and `int`/`unsigned int` with `d/u/x`